File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/info_bored_record_item.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class info_bored_record_item extends MsgBase
  {
      public info_bored_record_item()
      {
          CodeId = MsgCodeId.info_bored_record_item;
      }
      public String id;    //
      public String headid;    //
      public String userid;    //
      public String recordurl;    //
      public int recordlength;    //

}

File: server/hudie/hudie/message/Protocols/jie_gou/info_bored_record_item.cs
using messages;

namespace messages.Protocols
{
  public class info_bored_record_item : MsgBase
  {
      public info_bored_record_item()
      {
          CodeId = MsgCodeId.info_bored_record_item;
      }
      public string id;    //
      public string headid;    //
      public string userid;    //
      public string recordurl;    //
      public int recordlength;    //

	}
}
